zntrack/utils/config.py

```diff
@@ -18,20 +18,45 @@ class Config:
 
     Attributes
     ----------
-    debug: bool
-        TODO
     nb_name: str
         Name of the JupyterNotebook, if the Nodes are defined in a Notebook
     nb_class_path: Path
         The path where jupyter notebooks should write the *.py
-    no_dvc: bool, default = False
-        Do not write a dvc file when true.
     """
 
-    debug: bool = False  # not implemented yet
     nb_name: str = None
     nb_class_path: Path = Path("src")
-    no_dvc: bool = False
+
+
+@dataclass(frozen=True)
+class Files:
+    """Important File paths for ZnTrack to work
+
+    Notes
+    ------
+    Currently frozen because changing the value is not tested.
+    """
+
+    zntrack: Path = Path("zntrack.json")
+    params: Path = Path("params.yaml")
+
+
+@dataclass(frozen=True)
+class ZnTypes:
+    """Collection of ZnTrack Types to identify descriptors beyond their dvc option
+
+    Attributes
+    ----------
+    results: most zn.<options> like zn.outs() / zn.metrics() use this one
```

Comment:
I agree and I want to update the ZnTypes eventually. For now they aren't any different to just some string wrappers.
The reason results is in the docs is, because it is the first one that has been refactored a bit. I'm not particulalry happy with this part of ZnTrack but I haven't found a way around it yet.

Reply:
okay that is reasonable.